src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Email/Blocks/smtp_configuration.html.twig

```diff
@@ -1,101 +1,87 @@
+      <div class="col-sm">
+        {{ form_errors(emailConfigurationForm.smtp_config.server) }}
+        {{ form_widget(emailConfigurationForm.smtp_config.server) }}
+      </div>
+    </div>
 
-          <div class="form-group row">
-            <label class="form-control-label">{{ 'Encryption'|trans }}</label>
-            <div class="col-sm">
-              {{ form_errors(emailConfigurationForm.smtp_config.encryption) }}
-              {{ form_widget(emailConfigurationForm.smtp_config.encryption) }}
-              {% if not isOpenSslExtensionLoaded %}
-                <small class="form-text">
-                  {{ 'SSL does not seem to be available on your server.'|trans({}, 'Admin.Advparameters.Notification') }}
-                </small>
-              {% endif %}
-            </div>
-          </div>
+    <div class="form-group row">
+      {{ ps.label_with_help(('SMTP username'|trans), ('Leave blank if not applicable.'|trans({}, 'Admin.Advparameters.Help'))) }}
+      <div class="col-sm">
+        {{ form_errors(emailConfigurationForm.smtp_config.username) }}
+        {{ form_widget(emailConfigurationForm.smtp_config.username) }}
+      </div>
+    </div>
 
-          <div class="form-group row">
-            {{ ps.label_with_help(('Port'|trans), ('Port number to use.'|trans)) }}
-            <div class="col-sm">
-              {{ form_errors(emailConfigurationForm.smtp_config.port) }}
-              {{ form_widget(emailConfigurationForm.smtp_config.port) }}
-            </div>
-          </div>
+    <div class="form-group row">
+      {{ ps.label_with_help(('SMTP password'|trans), ('Leave blank if not applicable.'|trans({}, 'Admin.Advparameters.Help'))) }}
+      <div class="col-sm">
+        {{ form_errors(emailConfigurationForm.smtp_config.password) }}
+        {{ form_widget(emailConfigurationForm.smtp_config.password) }}
+      </div>
+    </div>
 
-          {% block smtp_configuration_form_rest %}
-            {{ form_rest(emailConfigurationForm.smtp_config) }}
-          {% endblock %}
-        </div>
+    <div class="form-group row">
+      <label class="form-control-label">{{ 'Encryption'|trans }}</label>
+      <div class="col-sm">
+        {{ form_errors(emailConfigurationForm.smtp_config.encryption) }}
+        {{ form_widget(emailConfigurationForm.smtp_config.encryption) }}
+        {% if not isOpenSslExtensionLoaded %}
+          <small class="form-text">
+            {{ 'SSL does not seem to be available on your server.'|trans({}, 'Admin.Advparameters.Notification') }}
+          </small>
+        {% endif %}
       </div>
-      <div class="card-footer">
-        <div class="d-flex justify-content-end">
-          <button class="btn btn-primary">{{ 'Save'|trans({}, 'Admin.Actions') }}</button>
-        </div>
+    </div>
+
+    <div class="form-group row">
+      {{ ps.label_with_help(('Port'|trans), ('Port number to use.'|trans)) }}
```

Comment:
for some texts in this page, I don't see translation domain as usual

Reply:
It was already like that in the previous twig template. AFAIK, it used the default domain name.